### PR TITLE
Hide "Translate" menu button if deepL is not configured.

### PR DESCRIPTION
**Pull request by gokaysatir**
Signed-off-by: Gökay Şatır <gokaysatir@collabora.com>
Change-Id: I52a02ab8380468b5dc56cb90a4d8edf2fab56c06

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: coolwsd.xml.in [hunk 1]
```diff
@@ -23,6 +23,7 @@
     </languagetool>
 
     <deepl desc="DeepL API settings for translation service">
+		<enabled desc="If DeepL translation settings are configured." default="false">false</enabled>
```
Comment (pedropintosilva) [inline coolwsd.xml.in:26]: there are two tabs here. It should be spaces

File: wsd/COOLWSD.cpp [hunk 2]
```diff
@@ -1973,6 +1973,7 @@ void COOLWSD::innerInitialize(Application& self)
         { "languagetool.ssl_verification", "true"},
         { "deepl.api_url", ""},
         { "deepl.auth_key", ""},
+        { "deepl.enabled", "false"}
```
Comment (pedropintosilva) [inline wsd/COOLWSD.cpp:1976]: missing comma at the end (to be  consistent with the rest of the file)

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
Thanks @gokaysatir , I tested and it works :) but

File: coolwsd.xml.in (hunk 1 above)
Comment (pedropintosilva) [reply coolwsd.xml.in:26]: also probably better to rephrase it (it sounds as if we add API url and API key the icon will appear which is not true). Maybe better to have instead "Enable Deepl translation service" and maybe add the type ( type="bool")

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: coolwsd.xml.in (hunk 1 above)
Comment (gokaysatir) [reply coolwsd.xml.in]: Corrected the tabs and added type, thanks :) i think description is fine but..

**Review by pedropintosilva — APPROVED** [review]
I squashed those 2 commits (and removed the duplicated sign off)

**Comment by gokaysatir** [comment]
> I squashed those 2 commits (and removed the duplicated sign off)

Thank you very much @pedropintosilva.

**Pull request merged**